### PR TITLE
Improved LSD chunks (needs liblcf#347)

### PR DESCRIPTION
**Pull request by fmatthew5876**
Depends on: https://github.com/EasyRPG/liblcf/pull/347

### CODE REVIEW

File: src/game_picture.cpp [hunk 1]
```diff
@@ -186,37 +186,37 @@ void Game_Picture::Move(const MoveParams& params) {
 	// Note that data.effect_mode doesn't necessarily reflect the
 	// last effect set. Possible states are:
 	//
-	// * effect_mode == 0 && finish_effect == 0
+	// * effect_mode == 0 && finish_effect_power == 0
```
Comment (carstene1ns) [inline src/game_picture.cpp]: Still using magic numbers here, maybe update/rewrite the comment

Comment (fmatthew5876) [reply src/game_picture.cpp]: Fixed

**Comment by carstene1ns** [comment]
jenkins: test this please.

**Comment by carstene1ns** [comment]
jenkins: test this please

**Comment by Ghabry** [comment]
Jenkins: test this please

**Pull request merged**